projects/groovy-lang.org/package.yml

```diff
@@ -0,0 +1,45 @@
+distributable:
+  url: https://groovy.jfrog.io/artifactory/dist-release-local/groovy-zips/apache-groovy-binary-{{version}}.zip
+  strip-components: 1
+versions:
+  url: https://groovy.jfrog.io/artifactory/dist-release-local/groovy-zips/
+  match: /apache-groovy-binary-\d+\.\d+\.\d+\.zip/
+  strip:
+    - /^apache-groovy-binary-/
+    - /\.zip/
+warnings:
+  - vendored
+dependencies:
+  openjdk.org: '*'
+runtime:
+  env:
+    GROOVY_HOME: "{{prefix}}"
+    darwin/aarch64:
+      #FIXME Unable to locate a Java Runtime.
+      JAVA_HOME: "{{deps.openjdk.org.prefix}}"
```

Comment:
This definitely wouldn't be darwin/aarch64 specific. If it's resolved on one of your other branches, I'd just mark the PR that it depends on that PR.